### ISSUE TITLE
Polymer 2: Safari does not draw change in active state while button focused

### ISSUE THREAD

**Issue by balloob** [PolymerElements/paper-toggle-button#124]
### Description
I am running into a super weird issue with the paper-toggle-button running in Safari with Polymer 2.

When you click on the paper-toggle-button, the change in active state is not updated until the element loses focus.

Works fine in Polymer 1 in Safari.
Works fine with Polymer 2 in Chrome.

Repro linked below, also included a video because who doesn't like videos: https://www.youtube.com/watch?v=tLr5w8UROKI&feature=youtu.be

I found a workaround by blurring and refocussing the active element after a change. It can be enabled in my repro.

```js
    const fix = function() {
      document.querySelector('paper-toggle-button').addEventListener('change', () => {
        const el = document.activeElement;
        el.blur();
        el.focus();
      });
    }
```

### Expected outcome

When clicked, the button should immediately update.

### Actual outcome

The CSS is not updated until clicked away.

### Live Demo
Repro: http://jsbin.com/nifanotipo/1/edit?html,output
Another repro is on the [paper-toggle-page on webcomponents.org](https://www.webcomponents.org/element/PolymerElements/paper-toggle-button). Note that in this case, you should click off the element instead of focusing the code box, as that will cause the element to be reset.

### Steps to reproduce

1. Put a `paper-toggle-button` element in the page.
2. Open the page in Safari.
3. Click the `paper-toggle-button` element.
4. Observe nothing happening
5. Click away

### Browsers Affected

Safari 10.1.2. Don't have access to older Safari's.

**Comment by balloob** [comment]
Note that losing the focus triggers it to work again, but it's not the cause of the problem. When we programmatically update the state to set checked = true, I have to focus and blur the element to make it render the correct CSS.

**Comment by balloob** [comment]
Here is an updated repo that includes a button to toggle the toggle button programmatically: http://jsbin.com/peloxeqisi/edit?html,output

**Comment by balloob** [comment]
This seems to have been fixed in Safari 11.0
